test/SourceKit/CodeComplete/complete_checkdeps_ownfile.swift

```diff
@@ -0,0 +1,100 @@
+// BEGIN State1.swift
+import ClangFW
+import SwiftFW
+
+func foo(val: MyStruct) {
+   /*HERE*/
+}
+
+// BEGIN State2.swift
+import ClangFW
+import SwiftFW
+
+func foo(val: MyStruct) {
+    val./*HERE*/
+}
+
+// BEGIN DUMMY.swift
+
+// Checks that editing and saving the current file doesn't affect dependency checking.
+// REQUIRES: shell
+
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/Frameworks)
+// RUN: %empty-directory(%t/MyProject)
+
+// RUN: %{python} %utils/split_file.py -o %t %s
+
+// RUN: COMPILER_ARGS=( \
+// RUN:   -target %target-triple \
+// RUN:   -module-name MyProject \
+// RUN:   -F %t/Frameworks \
+// RUN:   -I %t/MyProject \
+// RUN:   -import-objc-header %t/MyProject/Bridging.h \
+// RUN:   %t/MyProject/Library.swift \
+// RUN:   %t/test.swift \
+// RUN: )
+// RUN: INPUT_DIR=%S/Inputs/checkdeps
+// RUN: DEPCHECK_INTERVAL=1
+// RUN: SLEEP_TIME=2
```

Comment:
Is it intentional that you are not using `SLEEP_TIME` and `DEPCHECK_INTERVAL` in the test invocations?

Reply:
No. Fixed